docker-compose.yml

```diff
@@ -0,0 +1,37 @@
+networks:
+  overlay:
+
+services:
+  mysql:
+    image: mysql:8-debian
+    restart: always
+    env_file: docker-compose/mysql.env
+    volumes:
+      - mysql-db:/var/lib/mysql
+      - ./db_structure.sql:/docker-entrypoint-initdb.d/01-schema.sql
+      - ./seeds.sql:/docker-entrypoint-initdb.d/02-initdata.sql
+    networks:
+      - overlay
+    # TODO remove the next line when we support the new auth plugin
+    command: ["--default-authentication-plugin", "mysql_native_password"]
+
+  app:
+    depends_on: ["mysql"]
+    build:
+      context: .
+      target: development
+    command: ["npm", "start"]
+    volumes:
+      # Note: only files in src will be updated; for all code outside src we must rebuild the image
+      - ./src:/app/src
+    init: true
+    env_file:
+      - docker-compose/app.env
+    networks:
+      - overlay
+    ports:
+      - 8080:8080
+
+
```

Comment:
If you do want to include shibboleth services here, I would prefer if the environment variables were not included. Supporting files must not be included, e.g. database initialisation scripts. This could be done with a common `env_file` for all the shibboleth services that is not included or a `.env` file in the same folder that is not committed. I don't might including mapping environment variables with `environment`.